### PR TITLE
Set NewRelic appName

### PR DESCRIPTION
**Pull request by arnaudcharmier**

### CODE REVIEW

File: Routing/Filter/NewRelicFilter.php [hunk 1]
```diff
@@ -63,5 +69,13 @@ public function nameTransaction($name) {
 	public function hasNewRelic() {
 		return extension_loaded('newrelic');
 	}
+/**
+ * Set NewRelic AppName
+ *
```
Comment (jeremyharris) [inline Routing/Filter/NewRelicFilter.php:74]: Name param should be documented

**Comment by jeremyharris** [comment]
Nice change, mind adding a bit in the readme about the config key to document this feature?

File: Routing/Filter/NewRelicFilter.php (hunk 1 above)
Comment (arnaudcharmier) [inline Routing/Filter/NewRelicFilter.php:74]: Done ;)

**Comment by jeremyharris** [comment]
Great, thanks!

**Pull request merged**